cmd/rook/ceph/cleanup.go

```diff
@@ -63,6 +80,19 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 		}
 		// Remove all the mon directories.
 		cleanMonDirs()
+
```

Comment:
Instead of using the same [deleteDataDirOnHosts](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/cluster.yaml#L94) setting, the plan was to add a new setting to the cluster CR such as `wipeDevicesOnHosts=yes-really-destroy-data`.  The existing setting only indicates to delete the dataDirHostPath.

Reply:
Well, if the mons are gone, why would we have the OSD disks intact? It seems a bit overkill at this point.